content/integrations/rsyslog.md

```diff
@@ -1,9 +1,28 @@
 ---
-title: Rsyslog Log collection
-kind: Documentation
+title: Rsyslog
+kind: integration
 description: "Configure Rsyslog to gather logs from your host, containers & services."
+short_description: "Configure Rsyslog to gather logs from your host, containers & services."
```

Comment:
Serial comma here and in the line above. Configure Rsyslog to gather logs from your host, containers, & services.